kratos/solving_strategies/builder_and_solvers/p_multigrid/augmented_lagrange_constraint_assembler.cpp

```diff
@@ -129,10 +137,10 @@ struct AugmentedLagrangeConstraintAssembler<TSparse,TDense>::Impl
     using Interface = AugmentedLagrangeConstraintAssembler<TSparse,TDense>;
 
     /// @brief A map associating slave IDs with constraint indices and the number of constraint objects referencing it.
-    std::unordered_map<std::size_t,         //< identifier of the constraint equation (slave ID for MasterSlaveConstraint, CONSTRAINT_LABEL for MultifreedomConstraint)
-                       std::pair<
-                            std::size_t,    //< row index of the constraint equation in the relation matrix
-                            std::size_t     //< number of constraint objects defining the equation
+    boost::unordered_flat_map<std::size_t,         //< identifier of the constraint equation (slave ID for MasterSlaveConstraint, CONSTRAINT_LABEL for MultifreedomConstraint)
```

Comment:
Add a TODO to eventually replace this

Reply:
wat? Where do you mean?

Reply:
Instead of directly call the boost::unordered_flat_map, eventually that should be a Kratos::unordered_map